picasso/src/main/java/com/squareup/picasso/Cache.java

```diff
@@ -24,7 +24,7 @@
  * your {@link Cache} implementation is thread safe when {@link Cache#get(String)} or {@link
  * Cache#set(String, android.graphics.Bitmap)} is called.
  */
-public interface Cache {
+interface Cache {
```

Comment:
Can we lose this in a follow-up and rename `LruCache` to `Cache`?

Reply:
Yes, that's exactly what I was thinking.